csharp/HOLMS.Platform/HOLMS.Platform/Types/Extensions/IAM/Department.cs

```diff
@@ -1,45 +1,29 @@
 ﻿namespace HOLMS.Types.IAM {
     public partial class Department {
-        public Department(DepartmentIndicator id, string name, bool isHousekeeper, bool capStaffMgmt, bool capGroupMgmt,
-            bool capHousekeepingMgmt, bool capAccountingReports, bool capManagementReports,
-            bool capWorkstationParameters, bool capSaveDisplaySettings, bool capLodgeicReset,
-            bool capEventsWaitlistSetup, bool capSystemSettings, bool capYieldManagement,
-            bool capHoldMgmt, bool capGuestBlacklistAddRemove, bool capOverbookRooms,
-            bool capOverrideRates, bool capCallAccountingAdvancedSetup, bool capGiftShopsalesSetup,
-            bool capCreditCardSetup, bool capCreditCardWorksheet, bool capCreditCardCaptureSettle,
-            bool capCreditCardReporting, bool capExpediaQCMgmt, bool capWebSetup) {
+        public Department(DepartmentIndicator id, string name, bool isHousekeeper, bool capGetSupplySnapshot, bool capGetRoomTypes,
```

Comment:
In fact, you might want to develop some kind of platform-specific object like "claims" that could act as a bag of these things, that would be easily tested to see whether one is in it. Maybe a dict under the covers, but it should expose a single method, `Authorizes` that takes a security action, and returns true/false.